aws/templates/setContext.ftl

```diff
@@ -166,6 +166,7 @@
 [#if blueprintObject.Solution?has_content]
     [#assign solutionObject = blueprintObject.Solution]
     [#assign solnMultiAZ = solutionObject.MultiAZ!(environmentObject.MultiAZ)!false]
+    [#assign RDSAutoMinorVersionUpgrade = (segmentObject.RDS.AutoMinorVersionUpgrade)!(solutionObject.RDS.AutoMinorVersionUpgrade)!(environmentObject.RDS.AutoMinorVersionUpgrade)!true]
```

Comment:
I don't think this belongs here - we are moving stuff out of here - don't want to be adding things right now. Better to be in solution_rds via component configuration.